website/docs/Examples/AutoML-NLP.md

```diff
@@ -86,4 +86,43 @@ automl_settings["custom_hpo_args"] = {
 automl.fit(
     X_train=X_train, y_train=y_train, X_val=X_val, y_val=y_val, **automl_settings
 )
+```
+
+### A simple summarization example
```

Comment:
Have you tested this example locally?
If possible, display some example output like in other examples.